gammapy/estimators/points/sed.py

```diff
@@ -85,6 +85,29 @@ class FluxPointsEstimator(FluxEstimator, parallel.ParallelMixin):
     Notes
     -----
     For further explanation, see :ref:`estimators`.
+
+    Examples
+    --------
+    .. testcode::
+
+        from astropy import units as u
+        from gammapy.datasets import SpectrumDatasetOnOff
+        from gammapy.estimators import FluxPointsEstimator
+        from gammapy.modeling.models import PowerLawSpectralModel, SkyModel
+
+        path = "$GAMMAPY_DATA/joint-crab/spectra/hess/"
+        dataset = SpectrumDatasetOnOff.read(path + "pha_obs23523.fits")
+
+        pwl = PowerLawSpectralModel(index=2, amplitude='1e-12  cm-2 s-1 TeV-1')
+
+        dataset.models = SkyModel(spectral_model=pwl, name="crab")
```

Comment:
I have an issue with this example because it uses a spectrum that is very different from the best fit one on this dataset. 
While it might be working fine here, I don't think it will always be the case.